src/progress/pct/compile.p

```diff
@@ -81,6 +81,7 @@ DEFINE STREAM sWarnings.
 
 /* PCTCompile attributes */
 DEFINE VARIABLE OutputDir AS CHARACTER  NO-UNDO.
+DEFINE VARIABLE DestDir AS CHARACTER  NO-UNDO INITIAL ?.
```

Comment:
Just wondering, why don't you re-use outputDir variable ?

Reply:
I started out using OutputDir but a number of variables are derived from OutputDir.
The change was a lot smaller and IMHO less likely to introduce errors if I left OutputDir as a valid directory.

Revisiting the code I see that I probably missed 

    /* Does .r file exists ? */
    ASSIGN RCodeTS = getTimeStampDF(OutputDir, ipOutFile).

Reply:
> The change was a lot smaller and IMHO less likely to introduce errors if I left OutputDir as a valid directory.

Good idea

> Revisiting the code I see that I probably missed...

Missed what ?